src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -381,4 +394,11 @@ private boolean hasAllQualifiers(Feature feature, ConversionEntry conversionEntr
 
         return matchesAllQualifiers;
     }
+
+    public static class UnmappedFFFeature extends ValidationError {
```

Comment:
Rename UnmappedFFFeature -> UnmappedFFFeatureException
Rename ValidationError -> ValidationException (this is not an error in case of sevirity is WARN/OFF)